odoo/addons/base/models/assetsbundle.py

```diff
@@ -1085,3 +1174,30 @@ def get_command(self):
         except IOError:
             lessc = 'lessc'
         return [lessc, '-', '--no-js', '--no-color']
+
+
+class BinaryAsset(WebAsset):
+    def __init__(self, bundle, definition_bundle, **kwargs):
+        super().__init__(bundle, **kwargs)
+        self.extension = self._filename.split('.')[-1]
+        self.definition_bundle = definition_bundle
+
+    def _fetch_content(self):
+        try:
+            self.stat()
+            if self._filename:
+                with closing(file_open(self._filename, 'rb', filter_ext=BINARY_EXTENSIONS)) as fp:
```

Comment:
```suggestion
                with file_open(self._filename, 'rb', filter_ext=BINARY_EXTENSIONS) as fp:
```
redundant with file_open's `__exit__`